README.md

```diff
@@ -332,7 +332,7 @@ kover {
     intellijEngineVersion.set("1.0.656")    // change version of IntelliJ agent and reporter
     jacocoEngineVersion.set("0.8.7")        // change version of JaCoCo agent and reporter
     generateReportOnCheck = true            // false to do not execute `koverMergedReport` task before `check` task
-    disabledProjects = setOf()              // setOf("project-name") to disable coverage for project with name `project-name`
+    disabledProjects = setOf()              // setOf(":project-name") to disable coverage for project with path `:project-name` (`:` for the root project)
```

Comment:
It seems like an additional burden (apart from the fact that in **minor** release we'll break every configured project) for users to put `:` before the project name.

I suggest another approach:

* Allow using **both** short names and FQ names
* When using short name, search for projects with the same short name and refuse to configure the plugin unless unambigous name is specified

Reply:
implemented